rust/cymbal/src/langs/js.rs

```diff
@@ -229,13 +229,23 @@ impl From<(&RawJSFrame, JsResolveErr, &FrameLocation)> for Frame {
 impl From<&RawJSFrame> for Frame {
     fn from(raw_frame: &RawJSFrame) -> Self {
         metrics::counter!(FRAME_NOT_RESOLVED, "lang" => "javascript").increment(1);
+
+        // If this is a source_url: <anonymous> frame, we always assume it's not in_app
+        let is_anon = raw_frame
+            .source_url
+            .as_ref()
+            .map(|s| s == "<anonymous>")
+            .unwrap_or_default();
+
+        let in_app = raw_frame.in_app && !is_anon;
```

Comment:
Should we be doing this within the client? I'd rather not muddy cymbal by overriding the `in_app` value provided

Reply:
I'm fairly ok with cymbal acquiring a growing list of language specific hacks, so long as they remain consistently in the language specific modules and functions. I worry about relying on clients as the source of truth for this stuff - we can force-deploy a new version of cymbal, it's harder to forcibly upgrade all clients everywhere, as far as I know.

Reply:
Yeah, good point. For the same reasons harder to take anything we put in clients back